index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apple Page</title>
+    <script
+      src="https://kit.fontawesome.com/ad0c4e912f.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styler.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+    />
+  </head>
+  <body>
+    <div class="upper-section">
+      <div class="container">
+        <nav class="help-link-nav">
+          <a href="#"><span class="fa fa-apple"></span></a>
+          <a href="#">Store</a>
+          <a href="#">Mac</a>
+          <a href="#">iPod</a>
+          <a href="#">iPhone</a>
+          <a href="#">iTunes</a>
+          <a href="#">Support</a>
+          <div class="input-text-div"><input type="text" /></div>
+        </nav>
+        <div class="verse-div">
+            <a href="#">
+            <img src="images/verse-2.png" alt="verse-title"></a>
+        </div>
+        <div class="every-div">
+            <a href="#">
+            <img src="images/everyone.png" alt="everyone">
+        </a>
+        </div>
+
+        <a href="#" class="play-link"><p class="watch-par">
+          Watch the "Your Verse" film and explore the stories
+          <img src="images/play-img.png" alt="play"> 
+        </p></a>
+      </div>
+    </div>
+
+    <div class="lower-sec-div">
+        <div class="phone-type-div phone1">
+        </div>
+        <div class="phone-type-div phone2">
+        </div>
+        <div class="phone-type-div phone3">
+        </div>
+        <div class="phone-type-div phone4">
+        </div>
+    </div>
```

Comment:
Let's add a link to each article as in the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) one.